src/lib/fixers.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable no-console */
+const { request } = require('@octokit/request');
+
+async function inviteUser({ username, organization, token, role = 'member', dryRun = false }) {
+  console.log(`Setting ${username} up as a(n) ${role} for ${organization}.`);
+  if (dryRun) return;
+  // Do the thing!
+  const result = await request('PUT /orgs/{org}/memberships/{username}', {
+    headers: {
+      authorization: `token ${token}`
+    },
+    org: organization,
+    username: username,
+    role: role
+  });
+  if (result.status === 200) {
+    console.log(`${username} successfully invited to ${organization}!`);
+  } else {
+    console.log(`ERROR ERROR! ${username} could not be invited to ${organization}!`);
+  }
+}
+
+async function removeMember({ username, organization, token, dryRun = false }) {
+  console.log(`Removing ${username} from ${organization}.`);
+  if (dryRun === true) return;
+  // Do the thing!
+  const result = await request('DELETE /orgs/{org}/members/{username}', {
+    headers: {
+      authorization: `token ${token}`
+    },
+    org: organization,
+    username: username
+  });
+  if (result.status === 204) {
+    console.log(`${username} successfully removed from ${organization}.`);
```

Comment:
I'd suggest eventually passing down a logger object of some sort here so it's really easy to support things like `--silent` options from the cli. Not blocking for this PR though.

Reply:
Yeah I need to write up a ticket for proper output/logging. This was a really raw MVP, so everything is just a `console.log` for now.

Reply:
There's [this thing I wrote](https://github.com/winstonjs/winston) for that if you're into it